packages/vuetify/src/components/VIcon/VIcon.ts

```diff
@@ -202,9 +204,7 @@ export default Vue.extend({
 
     // Support usage of v-text and v-html
     if (data.domProps) {
-      iconName = data.domProps.textContent ||
-        data.domProps.innerHTML ||
-        iconName
+      iconName = data.domProps.textContent || data.domProps.innerHTML || iconName
```

Comment:
nit: eventho the linter doesn't complain, I feel like this can be reverted since it doesn't actually change anything